pkg/proc/breakpoints.go

```diff
@@ -195,6 +216,15 @@ func (bp *Breakpoint) CheckCondition(thread Thread) BreakpointState {
 	return bpstate
 }
 
+// checkHitCond evaluates bp's hit condition on thread.
+func (bp *Breakpoint) checkHitCond(bpstate BreakpointState, thread Thread) BreakpointState {
+	if bp.HitCond != nil {
+		bp.HitCond.X = astutil.Int(int64(bp.TotalHitCount))
+		bpstate.Active, bpstate.HitCondError = evalBreakpointCondition(thread, bp.HitCond)
```

Comment:
Reuse `bpstate.CondError`, only set it if it's already non-nil.

Reply:
Done